app/models/protocol.rb

```diff
@@ -460,35 +460,11 @@ def any_service_requests_to_display?
     return self.service_requests.detect { |sr| !['first_draft'].include?(sr.status) }
   end
 
-  def has_per_patient_per_visit? current_request, portal
+  def has_line_items_of_type? current_request, portal, one_time_fee = false
```

Comment:
I totally fail to see how that would enhance readability in the slightest. It's pretty simple: One time fee, or not. Also, switching over to strings instead of boolean would make the rest of the method less semantically consistent.